### DISCUSSION TITLE
Add classNames to the hostEl for Swiper element

### DISCUSSION THREAD

**Discussion by Ashkanph** [nolimits4web/swiper#7721] [Ideas]
Hello

I think it would be good to have the classNames for the hostEl (swiper-container) in Swiper element (WebComponent).  In v9 we had the classNames for the swiper-container. 

In my case, I need the `swiper-initialized` class to know when the Swiper is initialized. I know there are other approaches, such as using the `swiperafterinit` event, but having the classNames makes it easier.

If I am correct, the following changes will do it. I can create a PR if it is ok:

https://github.com/Ashkanph/swiper/commit/bf38ac3e8b9a84bd05bafd5544f240ac4701e374

**Comment by Ashkanph** [comment]
@nolimits4web, Could you please check the change to see if it is ok to create the PR? Thank you.

https://github.com/Ashkanph/swiper/commit/bf38ac3e8b9a84bd05bafd5544f240ac4701e374

**Comment by nolimits4web** [comment]
Hi, i would avoid spamming host element with the classes as in 99% of the cases they are not needed here and can lead to some unexpected behavior. I would stick with using afterinit/init events instead
